deploy/handler/operator.yaml

```diff
@@ -391,7 +395,7 @@ metadata:
   name: {{template "handlerPrefix" .}}nmstate-webhook
   namespace: {{ .HandlerNamespace }}
   annotations:
-    service.beta.openshift.io/serving-cert-secret-name: {{template "handlerPrefix" .}}nmstate-webhook
+    service.beta.openshift.io/serving-cert-secret-name: {{template "handlerPrefix" .}}openshift-nmstate-webhook
```

Comment:
This line also around `{{- if not .IsOpenShift }}`, I guess

Reply:
Not needed, this annotation is only used by openshift, so it's ok to have it there like this upstream.